### DISCUSSION TITLE
MAMBA F405 MINI MK3.5 - Outputs and Receiver configuration ok, but sticks can't send inputs

### DISCUSSION THREAD

**Discussion by Kabuby** [iNavFlight/inav#8426] [Q&A] [answered]
Dear all,
I come from Betaflight and I tried to install the 5.1 version of INAV on the MAMBA F405 MINI MK3.5 (Drone 4").
The target on the flight controller datasheet is MAMBAF405US_I2C.

It seems to me that I have configured everything correctly:
- calibrated accelerometers
- calibrated magnetometer
- GPS fix 12 satellites in my home
- Output + mixer seem ok and the motors are moving in the right direction
- channels in the receiver tab work fine
- telemetry on my Radiomaster ok

... but when I try to fly, after armed the drone, both sticks are not sending signals to the drone. Only if I try to throttle (more than half), all the motors accelerate quickly and it seems the drone goes crazy and I have to disarm.

Such a thing has never happened to me and I think I have something wrong somewhere.

Do you have any advice for me? ... just to try to do some tests?

Below the #diff all

Thanks in advance!

-----------------------------
# diff all

# version
# INAV/MAMBAF405US_I2C 5.1.0 Aug 19 2022 / 11:48:08 (76f22b25)
# GCC-10.2.1 20201103 (release)

# start the command batch
batch start

# reset configuration to default settings
defaults noreboot

# resources

# mixer

mmix reset

mmix 0  1.000 -1.000 -1.000  1.000
mmix 1  1.000 -1.000  1.000 -1.000
mmix 2  1.000  1.000 -1.000 -1.000
mmix 3  1.000  1.000  1.000  1.000

# servo mixer

# servo

# safehome

# feature
feature -TX_PROF_SEL
feature VBAT
feature GPS
feature CURRENT_METER
feature PWM_OUTPUT_ENABLE

# beeper

# blackbox
blackbox -NAV_ACC
blackbox NAV_POS
blackbox NAV_PID
blackbox MAG
blackbox ACC
blackbox ATTI
blackbox RC_DATA
blackbox RC_COMMAND
blackbox MOTORS

# map

# serial
serial 1 2 115200 115200 0 115200
serial 3 33554432 115200 115200 0 115200
serial 5 0 115200 115200 0 115200

# led

# color

# mode_color

# aux
aux 0 0 0 1700 2100
aux 1 51 1 1700 2100
aux 2 1 0 1700 2100
aux 3 11 5 1300 2100
aux 4 10 4 1300 2100
aux 5 52 2 1325 2100
aux 6 13 6 1300 2100

# adjrange

# rxrange
rxrange 0 988 2011
rxrange 1 988 2011
rxrange 2 988 2011
rxrange 3 988 2011

# temp_sensor

# wp
#wp 0 invalid

# osd_layout
osd_layout 0 0 33 0 V
osd_layout 0 1 12 0 H
osd_layout 0 7 22 12 H
osd_layout 0 9 37 6 V
osd_layout 0 11 2 6 V
osd_layout 0 12 1 7 V
osd_layout 0 14 16 0 V
osd_layout 0 20 0 16 V
osd_layout 0 21 0 17 V
osd_layout 0 22 22 6 V
osd_layout 0 23 26 0 V
osd_layout 0 28 36 8 V
osd_layout 0 30 10 13 V
osd_layout 0 32 12 0 H
osd_layout 0 34 18 2 V
osd_layout 0 36 2 8 V
osd_layout 0 54 8 0 V
osd_layout 0 96 1 1 V
osd_layout 0 109 38 2 V
osd_layout 0 110 38 0 V
osd_layout 0 111 19 6 H
osd_layout 0 112 38 1 V
osd_layout 0 120 2 2 V

# logic

# global vars

# programmable pid controllers

# master
set looptime = 500
set gyro_main_lpf_hz = 110
set gyro_main_lpf_type = PT1
set dynamic_gyro_notch_q = 250
set dynamic_gyro_notch_min_hz = 120
set setpoint_kalman_q = 200
set gyro_zero_x = -21
set gyro_zero_y = -11
set gyro_zero_z = -20
set ins_gravity_cmss =  963.255
set acc_hardware = MPU6000
set acczero_x = 12
set acczero_y = 14
set acczero_z = -110
set accgain_x = 4095
set accgain_y = 4111
set accgain_z = 4080
set align_mag = CW90
set mag_hardware = HMC5883
set magzero_x = -1046
set magzero_y = 775
set magzero_z = 273
set maggain_x = 615
set maggain_y = 553
set maggain_z = 520
set baro_hardware = BMP280
set receiver_type = SERIAL
set serialrx_provider = CRSF
set blackbox_rate_denom = 2
set motor_pwm_protocol = DSHOT600
set motor_poles = 12
set failsafe_procedure = DROP
set align_board_roll = 1800
set align_board_yaw = 1800
set current_meter_scale = 139
set bat_voltage_src = SAG_COMP
set model_preview_type = 3
set applied_defaults = 2
set gps_sbas_mode = AUTO
set gps_ublox_use_galileo = ON
set airmode_type = THROTTLE_THRESHOLD
set nav_user_control_mode = CRUISE
set osd_video_system = HD

# profile
profile 1

set mc_p_pitch = 44
set mc_i_pitch = 75
set mc_d_pitch = 25
set mc_i_roll = 60
set mc_p_yaw = 35
set mc_i_yaw = 80
set dterm_lpf_type = PT3
set d_boost_min =  0.800
set d_boost_max =  1.200
set antigravity_gain =  2.000
set antigravity_accelerator =  5.000
set smith_predictor_delay =  1.500
set tpa_rate = 20
set tpa_breakpoint = 1200
set rc_expo = 75
set rc_yaw_expo = 75
set roll_rate = 70
set pitch_rate = 70
set yaw_rate = 60

# profile
profile 2

# profile
profile 3

# battery_profile
battery_profile 1

set throttle_idle =  5.000

# battery_profile
battery_profile 2

# battery_profile
battery_profile 3

# restore original profile selection
profile 1
battery_profile 1

**Comment by JulioCesarMatias** [comment]
maybe it's the AirMode? try to disable it

**Comment by Kabuby** [comment]
Hi,
thanks for your reply! I tried to disable the "Permanent AIRMODE" and after arming the drone the Throttle channel works correctly, but the sticks on the Roll, Pitch and Yaw channel seem dead.

I am a bit frustrated, in the "Receiver" tab everything is moving correctly.

**Comment by MrD-RC** [comment, marked as answer]
I just sounds like you're expecting INAV to work like BetaFlight. It doesn't. Check out this issue https://github.com/iNavFlight/inav/issues/8413 I'm certain is answers your question too.

**Reply by Kabuby** [reply]
> I just sounds like you're expecting INAV to work like BetaFlight. It doesn't. Check out this issue #8413 I'm certain is answers your question too.

I took a test and everything works as it should!

Thanks a lot

**Comment by Kabuby** [comment]
Hi,
thank you for your answer!

Your observation below looks like just the problem I have described. I'll do some tests as soon as possible.

Thank you!

"So with motor stop off (correct for a quad), when you arm and enable AIRMODE the motors should spin, but your pitch, roll, and yaw inputs won't do anything. If you raise the throttle to above 15%, then lower it, the pitch, roll, and yaw sticks will have an effect. "
